tests/runner/src/env.rs

```diff
@@ -129,75 +126,20 @@ impl Display for ResultDisplayer {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match &self.result {
             Ok(result) => match result {
-                ObjectResult::Select(select_result) => {
-                    write!(
-                        f,
-                        "{}",
-                        SelectResultDisplayer {
-                            result: select_result
-                        }
-                        .display()
-                    )
-                }
                 ObjectResult::Mutate(mutate_result) => {
                     write!(f, "{mutate_result:?}")
                 }
-                // TODO(LFC): Implement it.
-                ObjectResult::FlightData(_) => unimplemented!(),
+                ObjectResult::FlightData(messages) => {
+                    let pretty = flight::flight_messages_to_recordbatches(messages.clone())
+                        .map_err(|e| e.to_string())
+                        .and_then(|x| x.pretty_print().map_err(|e| e.to_string()));
+                    match pretty {
+                        Ok(s) => write!(f, "{s}"),
+                        Err(e) => write!(f, "err: {e}"),
```

Comment:
```suggestion
                        Err(e) => write!(f, "format result error: {e}"),
```
Prefer to specify this error comes from formatting